[TypeScript] index.d.ts not copied under `node_modules/react-json-view/`

After pulling the latest version (1.16.2), I still don't see `index.d.ts` under `node_modules/react-json-view/`, nor is it under `dist`. Is there any steps missing? Could you look into this?

![image](https://user-images.githubusercontent.com/1334982/39439348-22e5d2ba-4c5c-11e8-97d4-f6bce319d846.png)

![image](https://user-images.githubusercontent.com/1334982/39439359-29a08550-4c5c-11e8-9a68-18cad0008d2f.png)

../../node_modules/react-json-view/index.d.ts(268,1): error TS1046: A 'declare' modifier is required for a top level declaration in a .d.ts file.

const ReactJson: React.ComponentType<ReactJsonViewProps>;
    ~~~~~

-----------------

line 267 should be:

declare const ReactJson: React.ComponentType<ReactJsonViewProps>;

--- a/index.d.ts
+++ b/index.d.ts
@@ -265,5 +265,5 @@ export type ThemeKeys =
   | 'tube'
   | 'twilight';
 
-const ReactJson: React.ComponentType<ReactJsonViewProps>;
+declare const ReactJson: React.ComponentType<ReactJsonViewProps>;
 export default ReactJson;

